edit/buildozer.go

```diff
@@ -573,6 +573,34 @@ func cmdDictRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return env.File, nil
 }
 
+// cmdDictListAdd adds a item to a list in a dict.
+func cmdDictListAdd(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	key := env.Args[1]
+	args := env.Args[2:]
+
+	dict := &build.DictExpr{}
+	currDict, ok := env.Rule.Attr(attr).(*build.DictExpr)
+	if ok {
```

Comment:
You can combine these two lines:

    if currDict, ok := env.Rule.Attr(attr).(*build.DictExpr); ok {

Reply:
Updated, thanks.